### PR TITLE
windows cordova plugin updates.

### PR DESCRIPTION
**Pull request by smcnulty-sfdc**

### CODE REVIEW

**Comment by bhariharan** [comment]
:+1:

**Comment by kchitalia** [comment]
:+1:

File: SalesforceSDK/CordovaPluginJavascript/com.salesforce.plugin.smartsyncProxy.js [hunk 1]
```diff
@@ -28,7 +28,8 @@ cordova.define("com.salesforce.SalesforceSmartStoreProxy", function(require, exp
      var SERVICE = "com.salesforce.smartstore";
```
Comment (kchitalia) [inline SalesforceSDK/CordovaPluginJavascript/com.salesforce.plugin.smartsyncProxy.js]: This should be smartsync, right?

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**